inginious/frontend/plugins/grader_generator/pages/graderforms.py

```diff
@@ -0,0 +1,130 @@
+import json
+from collections import OrderedDict
+from abc import ABC, abstractmethod
+from inginious.frontend.pages.course_admin.task_edit import CourseEditTask
+
+# TODO: Replace this class and all json dumps to main class only one time
```

Comment:
Is this TODO still true?

Reply:
No, I already did it. The error class was change and the json dumps are done one time 👍